src/components/layerlist/LayerLegendImage.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+    <!-- Remarks: 
+    As we need none of the responsive functionality of v-img, use a simple
+    HTLM img to stop the component from flickering, when the image is re-requested. 
```

Comment:
```suggestion
    HTML img to stop the component from flickering, when the image is re-requested. 
```

Reply:
Resolved in 5e2e02dacb459b74185d420090f8ce1c53e3fe31